test/display_tests/display.jl

```diff
@@ -81,58 +81,56 @@ let x, b
         @test string(a) == "1.5f0 ± 0.5f0"
     end
 
-    #= # TODO uncomment when Complex support is restored
     @testset "Complex{Interval}" begin
-        a = Complex(Interval(0, 2), 1)
+        a = Complex(Interval(0, 2), Interval(1))
         @test typeof(a) == Complex{Interval{Float64}}
         setformat(:standard)
-        @test string(a) == "[0, 2] + [1, 1]im"
+        @test string(a) == "[0.0, 2.0] + [1.0, 1.0]im"
 
         setformat(:midpoint)
-        @test string(a) == "(1 ± 1) + (1 ± 0)im"
+        @test string(a) == "(1.0 ± 1.0) + (1.0 ± 0.0)im"
     end
-    =#
 
     setprecision(BigFloat, 256)
 
     @testset "DecoratedInterval" begin
         a = @decorated(1, 2)
         @test typeof(a)== DecoratedInterval{Float64}
 
-        setformat(:standard, decorations=false)
-        @test string(a) == "[1, 2]"
+        setformat(:standard; decorations = false)
+        @test string(a) == "[1.0, 2.0]"
 
-        setformat(:standard, decorations=true)
-        @test string(a) == "[1, 2]_com"
+        setformat(; decorations = true)
+        @test string(a) == "[1.0, 2.0]_com"
 
         # issue 131:
         a = DecoratedInterval(big(2), big(3), com)
 
-        setformat(:standard, decorations=false)
-        @test string(a) == "[2, 3]₂₅₆"
+        setformat(; decorations = false)
+        @test string(a) == "[2.0, 3.0]₂₅₆"
 
-        setformat(decorations=true)
-        @test string(a) == "[2, 3]₂₅₆_com"
+        setformat(; decorations = true)
+        @test string(a) == "[2.0, 3.0]₂₅₆_com"
 
         setformat(:full)
         @test string(a) == "DecoratedInterval(Interval(2.0, 3.0), com)"
 
         setformat(:midpoint)
         @test string(a) == "2.5 ± 0.5_com"
 
-        setformat(decorations=false)
+        setformat(; decorations = false)
         @test string(a) == "2.5 ± 0.5"
 
     end
 
     setprecision(BigFloat, 128)
 
     @testset "BigFloat intervals" begin
```

Comment:
Unless I missed it, can you add the example from #316 to be sure it works now (and that it will continue to work).

Reply:
I added the requested tests.